app/javascript/packages/document-capture/components/acuant-capture.tsx

```diff
@@ -512,9 +512,19 @@ function AcuantCapture(
   }
 
   function onSelfieCaptureSuccess({ image }: { image: string }) {
+    const analyticsPayload: ImageAnalyticsPayload = getAddAttemptAnalyticsPayload({
+      width: null,
```

Comment:
Since `width`, `height`, and `fingerprint` are all null, why not omit those params from `getAddAttemptAnalyticsPayload`?

Reply:
@amirbey I'm good with making the fields optional. 👍🏻

Reply:
done :check